pipeline/scripts/13-excelExport.R

```diff
@@ -409,7 +409,12 @@ positivecontrol_list <- column_list[positive_controls_index]
 if (z_score == 1) {
   # find if one or more positive control samples are missing
   missing_pos <- c()
-  if (length(positivecontrol_list) != 3){
+  # any() grep because you get a vector of FALSE's and TRUE's. only one grep match is needed for each positive control
+  if (any(grep("^(P1002\\.)[[:digit:]]+_", positive_controls)) && 
+      any(grep("^(P1003\\.)[[:digit:]]+_", positive_controls)) && 
+      any(grep("^(P1005\\.)[[:digit:]]+_", positive_controls))){
+    cat("All three positive controls are present")
+  } else {
     missing_pos <- paste0(c("positive controls list is not complete: ", positivecontrol_list), collapse=" ")
```

Comment:
This comment should be clearer, as the user could interpret it the other way around. As if these are the missing pos controls.